dashboard/backend/main.go

```diff
@@ -1,17 +1,28 @@
 package main
 
 import (
+	"flag"
 	"log"
 	"net/http"
 	"os"
-
-	"flag"
+	"strconv"
 
 	"github.com/kubeflow/tf-operator/dashboard/backend/client"
 	"github.com/kubeflow/tf-operator/dashboard/backend/handler"
 )
 
+const (
+	DefaultFrontendDir string = "/opt/tensorflow_k8s/dashboard/frontend/build/"
+	DefaultBackendPort int    = 8080
+)
+
 func main() {
+	var frontendDir string
+	var port int
+	flag.StringVar(&frontendDir, "frontend-dir", DefaultFrontendDir,
```

Comment:
Please add the default location in the description.

`Directory of the dashboard frontend (Default to /opt/tensorflow_k8s/dashboard/frontend/build/)`